packages/webpack-cli/lib/utils/Compiler.js

```diff
@@ -108,10 +108,18 @@ class Compiler {
             this.compiler = await webpack(options);
             this.compilerOptions = options;
         } catch (err) {
+            // https://github.com/webpack/webpack/blob/master/lib/index.js#L267
+            // https://github.com/webpack/webpack/blob/v4.44.2/lib/webpack.js#L90
+            const ValidationError = webpack.ValidationError ? webpack.ValidationError : webpack.WebpackOptionsValidationError;
             process.stdout.write('\n');
-            logger.error(`${err.name}: ${err.message}`);
-            process.stdout.write('\n');
-            return;
+            // In case of schema errors print and exit process
+            // For webpack@4 and webpack@5
+            if (err instanceof ValidationError) {
+                logger.error(err.message);
+            } else {
+                logger.error(`${err.name}: ${err.message}`);
```

Comment:
Why do not use same logic here? `logger.error(err.message);`?

Reply:
I thought by old logic you meant to keep name before 

Then if else would be redundant 😄

Reply:
No need name of error :+1:

Reply:
Let's do it better: `logger.error(err);`, because error can have own `toString` and it can be different from `name` and `message`

Reply:
done